src/main/drivers/accgyro.h

```diff
@@ -23,6 +23,7 @@ typedef struct gyro_s {
     sensorInitFuncPtr init;                                 // initialize function
     sensorReadFuncPtr read;                                 // read 3 axis data function
     sensorReadFuncPtr temperature;                          // read temperature if available
+    sensorInterruptFuncPtr intStatus;
```

Comment:
I know that this is historical problem and you are following coding style, but it is probably better to write pointers to function as 
```
typedef void sensorInterruptFunc(bool *data);         // sensor read and align prototype

sensorInterruptFunc *intStatus;
```

C knows pointer type, there is no reason to hide it in typedef ...